app/templates/components/preprint-form-project-select.hbs

```diff
@@ -0,0 +1,38 @@
+<h3>Choose existing OSF project:</h3>
```

Comment:
Minor quibble, but heading numbers are usually intended to convey organization/ outline.  If we want a heading to be a specific size, we should prefer a css class over skipping heading levels. (eg, discourage starting at h3 just to get smaller text)

http://webaim.org/techniques/semanticstructure/#contentstructure
> Technically, lower degree headings should be contained within headings of the next highest degree (i.e., one should not skip heading levels, such as from an <h2> to an <h4>, going down the document).